src/mito2/src/cache/file_cache.rs

```diff
@@ -117,6 +117,7 @@ impl FileCache {
     }
 
     /// Reads a file from the cache.
+    #[allow(unused)]
```

Comment:
Maybe we can remove this method?

Reply:
Leave it to the time of refactoring. It’s a bit troublesome to modify unit tests.